source/standalone/workflows/ray/hyperparameter_tuning/vision_cfg.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pathlib
+import sys
+
+# Allow for import of items from the ray workflow.
+UTIL_DIR = pathlib.Path(__file__).parent.parent.parent
+sys.path.append(str(UTIL_DIR))
```

Comment:
This seems a little hacky, can you explain why this is needed?

Reply:
Yes. Since the standalone isn't set up as a python package, you need that to be to import the util that sets up the job cfg.  Could move the job cfg into this directory, but I think that makes it less clear. Down to move it though